routers/comments.py

```
@@ -0,0 +1,75 @@
+    updated_data = await comment_service.update_comment(comment_id, req, current_user["id"])
+
+    return CommonResponse(
+        status="success",
+        message="댓글을 성공적으로 수정하였습니다.",
+        data=updated_data
+    )
+
+
+@router.delete("/{comment_id}",status_code=status.HTTP_204_NO_CONTENT)
+async def delete_comment(
+        post_id: int,
```

Ruff (0.14.13)

Unused function argument: `post_id`

(ARG001)